src/Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs

```diff
@@ -430,7 +438,7 @@ void ISupportNuget.RegisterResolvedPackageReferences(IReadOnlyList<ResolvedPacka
             ScriptOptions = ScriptOptions.AddReferences(references);
         }
 
-        Task<PackageRestoreResult> ISupportNuget.RestoreAsync() => _packageRestoreContext.Value.RestoreAsync();
+        Task<PackageRestoreResult> ISupportNuget.RestoreAsync() => _packageRestoreContext.Value.RestoreAsync();        
```

Comment:
Nit: extra whitespace.

Reply:
Thanks for your feedback. Just installed "Trailing Whitespace Visualizer by Mads Kristensen" to make myself more whitespace-aware ;)

Reply:
reverted